### ISSUE TITLE
Crash when selecting custom adb

### ISSUE THREAD

**Issue by NicholasHong19** [Simizfo/WSATools#41]
crash when selecting custom adb

**Comment by Simizfo** [comment]
Please, I'd need more details to understand what happens.

**Comment by Diegorro98** [comment]
I was able to reproduce this issue:
1. Install WSATools
2. Install ADB separately if you haven't installed yet
3. Open WSATools, Settings, Custom ADB Location and select the installed ADB.
Then it will crash.
Also I tried to install successfully ADB and then an App, then I tried to select same adb that I installed with WSATools and it crash too.
I think that it crash always

**Comment by NicholasHong19** [comment]
yes its maybe a bug and the adb i use is from the android sdk website

**Comment by Simizfo** [comment]
Hmmm, this didn't happen before, maybe something changed. I'll look into this too!

**Comment by sirredbeard** [comment]
Can confirm.

WSA Tools 0.2.3.0
Windows Build 22621.675

Attempted uninstall and reset to same effect.

https://user-images.githubusercontent.com/33820650/197542480-91ec1723-c73c-44ab-bdc8-19cd13829501.mp4

If you let me know how to collect logs, I will.

**Comment by Simizfo** [comment]
I don't save any kind of logs, but that should be enough to reproduce it on my own. Thanks Hayden! Will try to fix this today

**Comment by Simizfo** [comment]
Reproduced. Working on a fix!

**Comment by Simizfo** [comment]
Anyone willing to try the new version before I push it on the Store? On both of my PCs WSA is dead, and even reinstalling doesn't fix it... ugh

Looks like the bug is fixed, but I still want to make sure APK installation works before pushing it to the Store

**Comment by Simizfo** [comment]
Fixed in 0.2.5 that is now released on the Microsoft Store, it might take some time to show in all markets. Thanks everyone for reporting! ❤️
